pandas/core/frame.py

```diff
@@ -4025,6 +4025,41 @@ def replace(
             method=method,
         )
 
+    def _replace_columnwise(
```

Comment:
i think may make sense to move this somewhere, maybe missing.py just to try to reduce size in this module. anyhow for another day.